package.json

```diff
@@ -20,6 +21,7 @@
     "pg": "4.1.1",
     "q": "1.1.2",
     "underscore": "1.7.0",
+    "underscore.string": "^3.0.1",
```

Comment:
Not sure if you want to remove the `^` semver range operator here. I don't think it does any harm since we're probably locking the version number via npm-shrinkwrap.json anyways.